karapace/config.py

```diff
@@ -62,6 +62,7 @@ class Config(TypedDict):
     sasl_mechanism: str | None
     sasl_plain_username: str | None
     sasl_plain_password: str | None
+    sasl_oauth_token: str | None
```

Comment:
This would require documentation in `README.md`. Is this option needed?

Reply:
Unfortunately yes. Just like the already existing `sasl_plain_username` and `sasl_plain_password`, which are likewise extracted from an auth header and not "proper" configuration parameters, and thus undocumented in the `README.md`.

These are needed to pass info eg. from `KafkaRest` to the `UserProxy`.